dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1318,6 +1319,12 @@ public Workflow manualRegister(@ApiParam(hidden = true) @Parameter(hidden = true
         newWorkflow.setWorkflowName(Strings.isNullOrEmpty(workflowName) ? null : workflowName);
         newWorkflow.setDefaultTestParameterFilePath(defaultTestParameterFilePath);
 
+        // check that the user should have access to this organization
+        final Set<String> organizations = sourceCodeRepo.getOrganizations();
+        if (!organizations.contains(newWorkflow.getOrganization())) {
```

Comment:
To reduce latency (and github calls), would prefer that this was a call to a method like `sourceCodeRepo.isMyOrganization(String org)` that applied the successive checks as needed (is the org my user name?  is org in my organizations?  is org in my non-org repos?).  But, I don't think that method exists anymore, so no change necessary.

Reply:
Agreed in general. 
In specific, this isn't called very much and we should be deprecating it over time as well.